var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -245,6 +245,12 @@ class Llvm(CMakePackage, CudaPackage):
     # Workaround for issue https://github.com/spack/spack/issues/18197
     patch('llvm7_intel.patch', when='@7 %intel@18.0.2,19.0.4')
 
+    # workaround for missing include: https://reviews.llvm.org/D89450
+    patch("limits_include.patch", when="@8:11.999")
```

Comment:
This patch has not actually been accepted into LLVM or am I reading thing the review incorrectly.

Reply:
You read it correctly, but it was accepted by google benchmark upstream.  The follow-on that this should actually be patched in more files (since the repo has more than one copy) is a concern though.

Reply:
I'm confused though, the repo contains only two copies of this file and the patch fixes both of them. Where is the third copy? The review mentions a "separate test-suite repo", so maybe llvm is not concerned?